### DISCUSSION TITLE
Missmatch between generated scss variables and framework variables

### DISCUSSION THREAD

**Discussion by d49017** [primefaces/community#1175] [Figma UI Kit]
Hi, I've a problem with the implementation of tokens into our project. I've a set of variables exported using Token Studio. I've converted it to _variables.scss file using Style Dictionary (and tokens-studio/sd-transforms) and here is the problem: variables produced by SD are different than variables used by PrimeVue in the theme-base directory.

So, instead of `$fontSize` I've got `$globalFontSize`, or instead of `$inputPadding` there are 4 inputPadding variables: `$inputPaddingTop`, `$inputPaddingRight`, `$inputPaddingBottom`, `$inputPaddingLeft`.

Can anybody help me with this?
I'm sure that the version of PrimeVue we're using is the same as the version of the theme (3.35.0), but I don't know the version of Figma UI Kit.

P.S. I've posted this question on discord at the beginning, later I found out about gh discussions.

**Comment by onursenture** [comment]
Hi, The tokens we use in Figma are implemented using _variables.scss, but they might not provide an exact match. This isn't a feature we offer for the current UI Kit version. However, an exact match could be possible with the new theming engine we are developing. This is an ongoing project, and we are still determining an estimated timeline for its release.

**Reply by d49017** [reply]
I see, so in that case what can I do to actually be able to use these tokens right now? Should I create a transformer that somehow converts the values to match what primevue uses, or is there another way?

**Reply by onursenture** [reply]
Yes, you can do that, but we do not currently offer official support for it.
